packages/commonwealth/server/routes/createReaction.ts

```diff
@@ -73,10 +70,10 @@ const createReaction = async (
       where: { id: thread_id },
     });
   } else if (comment_id) {
-    const root_id = (
+    const threadId = (
       await models.Comment.findOne({ where: { id: comment_id } })
-    ).root_id;
-    const comment_thread_id = root_id.substring(root_id.indexOf('_') + 1);
+    ).thread_id;
+    const comment_thread_id = threadId.substring(threadId.indexOf('_') + 1);
```

Comment:
Will this logic change? In general, does this file need updates on how it refers to threads?